ReactiveCocoa/WatchKit/WKInterfaceActivityRing.swift

```diff
@@ -0,0 +1,13 @@
+import ReactiveSwift
+import WatchKit
+import HealthKit
+
+@available(watchOSApplicationExtension 2.2, *)
+extension Reactive where Base: WKInterfaceActivityRing {
+	/// Sets the summary of the activity ring.
+	///
+	/// - Parameter animated: Whether updates are animated.
+	public func activitySummary(animated: Bool) -> BindingTarget<HKActivitySummary?> {
```

Comment:
Is it ok to use a computed property instead of a function, in order to allow users to customise the animation behaviour of the binding?

Reply:
Yeah, that makes sense. 👍 

We have a few existing `func`s that return `BindingTarget`s.